tests/test_config.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for Ralph's configuration loading
+"""
+
+from os import environ
+
+import ralph.defaults
+from ralph.defaults import config
+
+
+# pylint: disable=invalid-name, unused-argument
+def test_config_default(fs, monkeypatch):
+    """Test that when no value is given, the default parameter is chosen"""
+
+    def config_mock():
+        return {}
+
+    monkeypatch.setattr(ralph.defaults, "load_config", config_mock)
```

Comment:
You don't need to mock this as you are testing the default fallback. If the configuration file doesn't exist, it should work fine.

Reply:
Fixed